### DISCUSSION TITLE
How to modify the color of text and add text boxes?

### DISCUSSION THREAD

**Discussion by Alex-Yaf** [ShapeCrawler/ShapeCrawler#679] [Q&A]
Text can only be added within the shape, but the background color of the shape cannot be modified. The color of the text only has the get attribute and no set. I don't know how to modify the color of the text and add text boxes.

**Comment by AdamShakhabov** [comment]
The Wiki page was updated. Please check these links:

**Updating Shape Fill:** https://github.com/ShapeCrawler/ShapeCrawler/wiki/Examples#set-shape-fill
**Updating Text Color:** https://github.com/ShapeCrawler/ShapeCrawler/wiki/Examples#text
**Adding a New Shape:** https://github.com/ShapeCrawler/ShapeCrawler/wiki/Examples#add-shape
